R/super.R

```diff
@@ -1,11 +1,12 @@
 #' Force method dispatch to use a superclass
 #'
 #' @description
-#' `super()` causes the dispatch for the next generic to use the method for
-#' the specified superclass. It is useful when you want to implement a method
-#' in terms of the implementation of its superclass.
+#' `super(from, to)` causes the dispatch for the next generic to use the method
+#' for the superclass `to` instead of the actual class of `from`. It's needed
+#' when you want to implement a method in terms of the implementation of its
+#' superclass.
 #'
-#' # Compared to S3 and S4
+#' ## S3 & S4
 #' `super()` performs a similar role to [NextMethod()] in S3 or
 #' [methods::callNextMethod()] in S4, but is much more explicit:
 #'
```

Comment:
I'm reading the next bullet here

```
* The class that `super()` will dispatch to is known at the time you
write `super()`, not only when it's called.
```

The wording here after the comma seems slightly confusing, maybe

```
* The class that `super()` will dispatch to is known statically at the time you
write `super()`, as opposed to dynamically determining it at the time the generic
is called.
```